src/components/attribute-picker/attribute-picker.js

```diff
@@ -62,6 +62,9 @@ class AttributePicker extends ArrowKeysMixin(RtlMixin(LocalizeLabsElement(LitEle
 			/* When true, the user currently has focus within the input */
 			_inputFocused: { type: Boolean, reflect: false },
 
+			/* When true, the user has reached the limit of attributes that can be added */
+			_limitReached: { type: Boolean, reflect: false },
```

Comment:
Could use reactive state for this kind of thing.

```suggestion
			_limitReached: { state: true },
```